uint8 PASSIVE = 0
uint8 RATE = 1
uint8 TRACK = 2
uint8 CALI = 3

uint8 mode
float64 yaw_rate
float64 pitch_rate
float64 bullet_speed